service-spec/service-spec-commons/src/main/java/io/hyscale/servicespec/commons/exception/ServiceSpecErrorCodes.java

```diff
@@ -31,8 +31,8 @@
     ERROR_WHILE_FETCH_SERVICE_SPEC_FIELD("Failed to fetch service spec field {}"),
     INVALID_FILE_EXTENSION("Invalid file extension:{} given."),
     INVALID_FORMAT("Invalid format {}."),
-    MISSING_FIELD_IN_PROFILE_FILE("Cannot find field {},in the specified profile file.Required for fetching {}"),
-    MISSING_FIELD_IN_SERVICE_FILE("Cannot find field {},in the specified service file.Required for fetching {}"),
+    MISSING_FIELD_IN_PROFILE_FILE("Cannot find field {},in the specified profile file."),
+    MISSING_FIELD_IN_SERVICE_FILE("Cannot find field {},in the specified service file."),
```

Comment:
Cannot find {} in {} hspec/ hprof . Instead of specified give the hspec input option